### PR TITLE
blog: Add Five ways to pass parameters to Apache APISIX post

### PR DESCRIPTION
**Pull request by nfrankel**
Screenshot 2024-04-25 at 18 18 39

### CODE REVIEW

File: blog/en/blog/2024/05/02/pass-parameters-apisix.md [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+  - HTTP API
+description: >
+  I recently read 6 Ways To Pass Parameters to Spring REST API. Though the title is a bit misleading, as it's unrelated to REST, it does an excellent job listing all ways to send parameters to a Spring application. I want to do the same for Apache APISIX; it's beneficial when you write a custom plugin.
+tags: [Community]
+image: https://static.apiseven.com/uploads/2024/04/25/V05nSV5W_american-football-63109.jpg
+---
+
+<head>
+    <link rel="canonical" href="https://blog.frankel.ch/fix-duplicate-api-requests/" />
+</head>
```
Comment (pottekkat) [inline blog/en/blog/2024/05/02/pass-parameters-apisix.md]: Wrong URL?

File: blog/en/blog/2024/05/02/pass-parameters-apisix.md [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+    title: Author
+    url: https://github.com/nfrankel
+    image_url: https://avatars.githubusercontent.com/u/752258
+keywords:
+  - APISIX
+  - REST
+  - HTTP API
+description: >
+  I recently read 6 Ways To Pass Parameters to Spring REST API. Though the title is a bit misleading, as it's unrelated to REST, it does an excellent job listing all ways to send parameters to a Spring application. I want to do the same for Apache APISIX; it's beneficial when you write a custom plugin.
+tags: [Community]
```
Comment (pottekkat) [inline blog/en/blog/2024/05/02/pass-parameters-apisix.md]: Ecosystem or Plugins maybe better.

File: blog/en/blog/2024/05/02/pass-parameters-apisix.md [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+image: https://static.apiseven.com/uploads/2024/04/25/V05nSV5W_american-football-63109.jpg
+---
+
+<head>
+    <link rel="canonical" href="https://blog.frankel.ch/fix-duplicate-api-requests/" />
+</head>
+
+>I recently read [6 Ways To Pass Parameters to Spring REST API](https://javabulletin.substack.com/p/6-ways-to-pass-parameters-to-spring). Though the title is a bit misleading, as it's unrelated to REST, it does an excellent job listing all ways to send parameters to a Spring application. I want to do the same for Apache APISIX; it's beneficial when you write a custom plugin.
+
+<!--truncate-->
```
Comment (pottekkat) [inline blog/en/blog/2024/05/02/pass-parameters-apisix.md:24]: Did we fix the overflow issue?

File: blog/en/blog/2024/05/02/pass-parameters-apisix.md [hunk 4]
```diff
@@ -0,0 +1,296 @@
...
+
+The result is as expected with the same request as above:
+
+```
+n: 15, offset: 3
+```
+
+## Query parameters
+
+Query parameters are another regular way to pass data. Like path variables, you can only pass simple values, _e.g._, `/?foo=bar`. The Lua code doesn't require regexp:
```
Comment (kayx23) [inline blog/en/blog/2024/05/02/pass-parameters-apisix.md]: `regexp` -- > `RegEx` ?

File: blog/en/blog/2024/05/02/pass-parameters-apisix.md (hunk 1 above)
Comment (nfrankel) [reply blog/en/blog/2024/05/02/pass-parameters-apisix.md]: Good catch!

File: blog/en/blog/2024/05/02/pass-parameters-apisix.md (hunk 3 above)
Comment (nfrankel) [reply blog/en/blog/2024/05/02/pass-parameters-apisix.md:24]: No clue

File: blog/en/blog/2024/05/02/pass-parameters-apisix.md (hunk 4 above)
Comment (nfrankel) [reply blog/en/blog/2024/05/02/pass-parameters-apisix.md]: I'll use regular expressions to avoid issues.

**Review by kayx23 — APPROVED** [review]

**Review by pottekkat — APPROVED** [review]

**Comment by nfrankel** [comment]
Thanks @pottekkat

**Comment by pottekkat** [comment]
Forgot to hit merge 😅

**Pull request merged**